Split String - Eight Code

Ganeshbabu18 opened the pull request.

vrushalee123 reviewed (changes requested):
Please add readMe. Add Input and output of each program.

Ganeshbabu18 commented:
> Please add readMe. Add Input and output of each program.

I have added nput and output of the program. but how to add readMe.

Ganeshbabu18 commented:
> > Please add readMe. Add Input and output of each program.
> 
> I have added nput and output of the program. but how to add readMe.

I have added readMe file too
